src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden ? this.hidden = false : this.hidden = true;
```

Comment:
https://github.com/mate-academy/js_herbivores_and_carnivores/blob/master/checklist.md
```suggestion
    this.hidden = !this.hidden;
```